frontend/postcss.config.js

```diff
@@ -0,0 +1,6 @@
+export default {
+  plugins: {
+    tailwindcss: {},
+    autoprefixer: {},
+  },
+}
```

Comment:
Your changes shouldn't conain the changes from #14, please remove all of them and start this branch from the `main` branch.